dev/preview/previewctl/cmd/get_name.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright © 2022 NAME HERE <EMAIL ADDRESS>
+
+*/
```

Comment:
This is in a few places and is unnecessary :) Unless there's actually a requirement to fill it with something relevant.

Reply:
Oh whoops, that was auto-generated and I forgot to clean it up 😅. Thanks!

It is required indeed, but not like that :P